comment/messages.py

```diff
@@ -86,3 +88,13 @@ class FlagState:
 class FollowError:
     EMAIL_REQUIRED = _('Email is required to subscribe {model_object}')
     SYSTEM_NOT_ENABLED = _('Subscribe system must be enabled')
+
+
+class BlockState:
+    UNBLOCKED = _('Unblocked')
+    BLOCKED = _('Blocked')
+
+
+class BlockedUserMSG:
+    NOT_PERMITTED = _('You cannot perform this action at the moment! Contact the admin for more details')
```

Comment:
i feel this message should be customizable.

Reply:
suggestion?

Reply:
I think we should probably add this to defaults with this as its default value.

Reply:
hmmm, but this error message shows up only when the user is blocked!

Reply:
so suppose i want the message to be `you are blocked to comment from this website, please container someone@domain if you think this maybe a mistake`.
 
Some might want to not include the email thing.

Someone else might just want to say `couldn't process the comment` to have some sort of abstraction.

so when passing on this message, we can do something like `settings.COMMENT_RESPONSE_FOR_BLOCKED_USER`.

Reply:
Nice idea.
BTW, I was thinking of a way to allow devs overriding the messages.py module by creating their own module and we import in the end of our message.py.